docs/Components/AttachedComponent.md

```diff
@@ -60,6 +60,15 @@ plane.angle = math.radians(5)
 print(math.degrees(aileron.angle))
 >>> 14.99999999999
 ```
+## position property
+This property returns the position vector of the component's origin in relation to its parent's origin
```

Comment:
In fact it returns the position in the absolute space.
The way its written seems that no matter the parent's position, the child will always return the same, when in fact this would be the *relative_position* property.